sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/SyncInitialSetupActivity.kt

```diff
@@ -86,5 +87,15 @@ class SyncInitialSetupActivity : DuckDuckGoActivity() {
         binding.deviceNameTextView.text = viewState.deviceName
         binding.primaryKeyTextView.text = viewState.primaryKey
         binding.secretKeyTextView.text = viewState.secretKey
+        binding.connectedDevicesList.removeAllViews()
+        if (viewState.isSignedIn) {
```

Comment:
All this is a bit meh, but just for testing and demo.

Reply:
as long we clean it I don't mind for now.